test/export/StructureDefinitionExporter.test.ts

```diff
@@ -1200,6 +1200,122 @@ describe('StructureDefinitionExporter', () => {
     expect(loggerSpy.getLastMessage()).toMatch(/File: Fixed\.fsh.*Line: 4\D*/s);
   });
 
+  it.skip('should log an error when a fixed value on a child interferes with a fixed slice on the parent', () => {
```

Comment:
Should these tests for fixed value be in the "Rules with Slices" describe block?

Reply:
These tests can probably be removed, actually. They're leftovers from a previous implementation, and I've added `todo` tests for FixedValueRule up on lines 2761-2766. Since supporting that rule is going to be handled in a different task, there's no need for them to stick around.